### DISCUSSION TITLE
Lines are not immediately shown when very smooth

### DISCUSSION THREAD

**Discussion by unoffending** [opentoonz/opentoonz#6394] [General]
### Is your feature request related to a problem? Please describe

If, for example, you set the line alignment to 50, then the line itself is not shown immediately. This may interfere with drawing accuracy.

### Describe the solution you'd like

I would like the line to show right away, but it doesn't have to be smooth right away. For example, in Adobe Animate the line is visible immediately when you use a large alignment, it’s just not aligned at the time of drawing.

**Comment by RodneyBaker** [comment]
Related to #5489

**Comment by RodneyBaker** [comment]
Converting this to Discussion so others can explore available information and consider the best way forward.
